### PR TITLE
rdk-29131

### PR DESCRIPTION
**Pull request by mstrozyn**

### CODE REVIEW

**Comment by mstrozyn** [comment]
Verification build https://gerrit.teamccp.com/#/c/476390/

File: helpers/utils.h [hunk 1]
```diff
@@ -344,4 +349,40 @@ namespace Utils
     bool isPluginActivated(const char* callSign);
 
     bool getRFCConfig(char* paramName, RFC_ParamData_t& paramOutput);
+
+    struct Telemetry
+    {
+        static void init()
+        {
+#ifdef ENABLE_TELEMETRY_LOGGING
+            t2_init("Thunder_Plugins");
```
Comment (anand-ky) [inline helpers/utils.h:358]: Pls add a FindTelemetry.cmake under cmake folder to find the telemetry 2.0 header files and libraries. You can then add Find_package to all the places where LOGERR is used.

**Review by anand-ky — CHANGES_REQUESTED** [review]

File: helpers/utils.h (hunk 1 above)
Comment (mstrozyn) [reply helpers/utils.h:358]: Enabling, linking and compilation dependency is set in another review (part of the verification build) https://gerrit.teamccp.com/#/c/476390/1/recipes-extended/rdkservices/rdkservices_git.bb

Comment (anand-ky) [reply helpers/utils.h:358]: ok sounds good.

**Review by anand-ky — APPROVED** [review]

**Pull request merged**